src/core/prototypes/marks/line.ts

```diff
@@ -255,4 +256,53 @@ export class LineElementClass extends EmphasizableMarkClass<
       })
     ];
   }
+
+  public getTemplateParameters(): TemplateParameters {
+    const properties = [];
+    if (this.object.mappings.visible.type !== "scale") {
```

Comment:
Could `this.object.mappings.visible` be undefined/null here?  earlier you were checking if it was null/undefined.

Reply:
Fixed